packages/template-retail-react-app/app/components/list-menu/index.jsx

```diff
@@ -260,7 +259,7 @@ const ListMenu = ({maxColumns = MAXIMUM_NUMBER_COLUMNS}) => {
                     </Stack>
                 ) : (
                     <Center p="2">
-                        <Spinner opacity="0" size="lg" />
+                        <Spinner size="lg" />
```

Comment:
Do we want the spinner to show when there are no categories?

Reply:
Do we know if a cate has sub before fetching the detail? If we do, it makes sense to not show spinner since there is nothing to load.

Reply:
Can you clarify? This spinner is for when there are no level-0 categories.

Reply:
What I meant was spinner is an indicator for something is being fetched. If we know there is nothing to fetch before hands, there is no need to use spinner. Otherwise, we have to show spinner when the fetching is happening.